specutils/io/default_loaders/jwst_reader.py

```diff
@@ -22,52 +26,78 @@ def identify_jwst_fits(origin, *args, **kwargs):
             # This indicates the data product contains  spectral data
             if not 'EXTRACT1D' in hdulist:
                 return False
+            if not hdulist[0].header["TELESCOP"] == "JWST":
+                return False
         return True
     # This probably means we didn't have a FITS file
     except Exception:
         return False
 
 
-@data_loader("JWST", identifier=identify_jwst_fits, dtype=SpectrumList,
+@data_loader("JWST", identifier=identify_jwst_x1d_fits, dtype=SpectrumList,
```

Comment:
```suggestion
@data_loader("JWST_x1d", identifier=identify_jwst_x1d_fits, dtype=SpectrumList,
```

I think this is necessary if we want to have a whole family of JWST loaders, right? I.e., if we don't do this there's no way to use the name to specify *which* of the JWST loaders to use?

Reply:
Yes, I think this is necessary as well.  Would this be a backwards incompatible change though?  Does that matter?

Perhaps this is where my limited knowledge of the infrastructure of the io registry and specifically in how it has been implemented in `specutils` works.